src/components/Excerpt.tsx

```diff
@@ -154,19 +199,14 @@ export default function Excerpt({
   // expanding/collapsing the content.
   // prettier-ignore
   const isOverflowing = contentHeight > (collapsedHeight + overflowThreshold);
-  const isCollapsed = inlineControls ? collapsedByInlineControls : collapse;
+  const isCollapsed = inlineControl ? collapsedByInlineControls : collapsed;
```

Comment:
The test here needs to be whether `collapsed` is defined. Otherwise the combination of inline controls + external state won't work. This will also require changing how `collapsed` is initialized.

Reply:
Right! I'm going to follow the same approach we use for the shared `Checkbox` component: https://github.com/hypothesis/frontend-shared/blob/d62a57c55537a531343e3b18e07fa4548bdc75ad/src/components/input/Checkbox.tsx#L47-L52